src/main/resources/view/DarkTheme.css

```diff
@@ -493,8 +493,7 @@
 
 // ************* interview panel css *************
    .interview-panel-header {
-       -fx-background-color: derive(#353C51, 10%);
-   }
+    -fx-background-color: #3c3e3f;   }
```

Comment:
The bracket `}` at the end should probably be on the next line right? 🙈

Reply:
will make the change for this!